test/Cell_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/Cell'
+require './lib/Ship'
+
+require "pry"
```

Comment:
Great Grant! We can take this pry out  and binding.pry and we'll be good! Great way for me to learn how to comment lol